### ISSUE TITLE
Data valuation for CheXpert dataset

### ISSUE THREAD

**Issue by ffeiland** [sunblaze-ucb/data-valuation#2]
Hi there,

I would like to apply the data valuation methods (KNN-Shap, DShap, TMC-Shap, LOO-values) on the Chexpert dataset (https://stanfordmlgroup.github.io/competitions/chexpert/) in order to detect noisy labels. Therefore, the ResNet152 model should be fine as a baseline.
As I am quite new to machine learning I would like to ask for the best way to do this.
- How do the images (320x320 pixels) need to be pre-processed in order to run the calculations?
- How should the training and test data be split up?

Thanks in advance,

Regards,
Fabian

**Comment by ffeiland** [comment]
To be more precise, when running the code from 
> Use Case >> Noisy Label 
on the cifar dataset I get following error: **RuntimeError:** Expected 4-dimensional input for 4-dimensional weight [64, 3, 3, 3], but got 2-dimensional input of size [10, 3072] instead
This seems to happen when the run_methode is fitting the model.
Can anyone help with the RuntimeError?

Thanks in advance.

**Comment by ffeiland** [comment]
As a start, I tried to reproduce the results of the Paper "An Empirical and Comparative Analysis of Data Valuation with Scalable Algorithms" by running the src from the use_case folder on the CIFAR dataset. 
I discovered first problems by feeding the data into the network. There are two different **fit()-methods** (pytorch_fitmodule.py, Shapley.py) which seem to require different data sizes. 
When feeding the image as **3x32x32 pixels** into the network, knn-, loo- and tmc-calculations work just fine, but g-shapley-calculations don't. 
When feeding the image as flattened **3072 pixels** into the network, g-shapley-calculations work, but the rest doesn't.
Is it a general problem of the src or did I do anything wrong?

Thanks in advance!
Regards,
Fabian
